AdaptorsFunctionality/src/au/gov/asd/tac/constellation/functionality/adaptors/dataaccess/plugins/importing/ImportFromRDFPlugin.java

```diff
@@ -59,142 +65,140 @@
 @ServiceProviders({
     @ServiceProvider(service = DataAccessPlugin.class),
     @ServiceProvider(service = Plugin.class)})
-
 @PluginInfo(pluginType = PluginType.IMPORT, tags = {"IMPORT"})
 @NbBundle.Messages("ImportFromRDFPlugin=Import From RDF")
 public class ImportFromRDFPlugin extends RecordStoreQueryPlugin implements DataAccessPlugin {
 
-//    private static final Logger LOGGER = Logger.getLogger(RDFViewer.class.getName());
-    public static final String INPUT_FILE_PARAMETER_ID = PluginParameter.buildId(ImportFromRDFPlugin.class, "input_file");
+    private static final Logger LOGGER = Logger.getLogger(ImportFromRDFPlugin.class.getName());
+
+    // parameters
+    public static final String INPUT_FILE_URI_PARAMETER_ID = PluginParameter.buildId(ImportFromRDFPlugin.class, "input_file_uri");
+    public static final String INPUT_FILE_FORMAT_PARAMETER_ID = PluginParameter.buildId(ImportFromRDFPlugin.class, "input_file_format");
 
     @Override
     protected RecordStore query(RecordStore query, PluginInteraction interaction, PluginParameters parameters) throws InterruptedException, PluginException {
 
-        String inputFilename = parameters.getParameters().get(INPUT_FILE_PARAMETER_ID).getStringValue();
+        final String inputFilename = parameters.getParameters().get(INPUT_FILE_URI_PARAMETER_ID).getStringValue();
+        final String intpuFileFormat = parameters.getParameters().get(INPUT_FILE_FORMAT_PARAMETER_ID).getStringValue();
 
         //TODO Research RDF4J etc
         //TODO Research base predicates; RDFS standard and what they map to in consty
-        //TODO Seperate queries to retrieve those
+        //TODO Seperate queries to retrieve base predicates
         //TODO Develop ontology for constellation -> mapping RDF stuff to existing constellation stuff (Allow for icons etc) <BASIC MAPPING>
         //TODO Potentially: Seperate query for mapping from RDF to Consty <SPECIFIC MAPPING>
         //TODO Add triples to constellation graph and update display; RDF view?
-        HashMap<String, String> prefixes = new HashMap<>();
-        prefixes.put("country", "http://eulersharp.sourceforge.net/2003/03swap/countries#");
-        prefixes.put("foaf", "http://xmlns.com/foaf/0.1/");
-        prefixes.put("jur", "http://sweet.jpl.nasa.gov/2.3/humanJurisdiction.owl#");
-        prefixes.put("dce", "http://purl.org/dc/elements/1.1/");
-        prefixes.put("dct", "http://purl.org/dc/terms/");
-        prefixes.put("owl", "http://www.w3.org/2002/07/owl#");
-        prefixes.put("rdfs", "http://www.w3.org/2000/01/rdf-schema#");
-        prefixes.put("skos", "http://www.w3.org/2004/02/skos/core#");
-        prefixes.put("music", "http://neo4j.com/voc/music#");
-        prefixes.put("ind", "http://neo4j.com/indiv#");
```

Comment:
FYI you can get this by res.getNamespaces()

Reply:
Agree. We can get rid of these lines.